server/monitor_test.go

```diff
@@ -59,6 +59,18 @@ func runMonitorServer() *Server {
 	return RunServer(opts)
 }
 
+func runMonitorServerWithAccounts() *Server {
+	resetPreviousHTTPConnections()
+	opts := DefaultMonitorOptions()
+	aA := &Account{Name: "A"}
```

Comment:
Would call `NewAccount("A")` here so that account is created with its default settings, etc.. Same for account B obviously.

Reply:
fixed